src/components/Bookcard.js

```diff
@@ -1,41 +1,68 @@
-import React from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getBooks, removeBook } from '../redux/books/books';
+import progressBar from '../Assets/progress.png';
+import '../styles/book.css';
 
-export default function bookcard() {
+function Bookcard() {
   const bookStore = useSelector((state) => state.booksReducer);
   const dispatch = useDispatch();
 
+  useEffect(() => {
+    if (!bookStore.length) {
+      dispatch(getBooks());
+    }
+  }, []);
+
   const handleRemove = (id) => {
     dispatch(removeBook(id));
   };
 
+  const percentage = Math.floor(Math.random() * 99);
+
   return (
     <div>
-      {
-          bookStore.map(({ id, author, title }) => (
-            <div key={id} className="card-container">
-              <p className="book-category">Science Fiction</p>
-              <h3>{title}</h3>
-              <p>{author}</p>
-              <p>Comments</p>
-              <span />
-              <button type="button" onClick={() => handleRemove(id)}>Delete</button>
-              <span />
-              <p>Edit</p>
-              <span />
-              <div className="progress-circle">
-                <span>8%</span>
-                <p>Completed</p>
-              </div>
-              <hr />
-              <p>CURRENT CHAPTER</p>
-              <p>Chapter 3: A Lesson Learned</p>
-              <button type="submit">UPDATE PROGRESS</button>
+      {bookStore.map(({
+        itemId, category, title,
+      }) => (
+        <li key={itemId} className="book-container d-flex">
+          <div className="general-info d-flex">
+            <span className="genre">{}</span>
+            <h3 className="title">{title}</h3>
+            <span className="author">{category}</span>
+            <ul className="book-management d-flex">
+              <li><button type="button">Comments</button></li>
+              <li>
+                {' '}
+                <button type="button" onClick={() => handleRemove(itemId)}>Delete</button>
+              </li>
+              <li><button type="button">Edit</button></li>
```

Comment:
- [ ] Kindly remove the `background-color ` you added to the `Comment` `Delete` and `Edit` buttons. As required the styling should EXACTLY be the same as the one provided on [Zeplin](https://zeplin.io/)

|---Figma----|---------Yours--------|
|---------------|-------------------------|
![go](https://user-images.githubusercontent.com/57562869/156876980-84fff815-f057-4bce-a854-8361b8a71d78.png)|![bot](https://user-images.githubusercontent.com/57562869/156876987-4e3df975-76d1-41ed-8ad5-c778f27b24aa.png)

Reply:
Thanks done!